kroxylicious-operator/examples/simple/01.KafkaProxy.simple.yaml

```diff
@@ -14,4 +14,8 @@ spec:
   clusters:
     - name: "my-cluster"
       upstream:
-        bootstrapServers: my-cluster-kafka-bootstrap.kafka.svc.cluster.local:9092
\ No newline at end of file
+        bootstrapServers: my-cluster-kafka-bootstrap.kafka.svc.cluster.local:9092
+      filters:
```

Comment:
the readme for `simple` says it justs forwards without doing any filtering, I'm guessing this is unintentional? Currently applying `examples/simple` results in the pod failing to start because virtualClusters is null.